release-notes/releases-index.json

```diff
@@ -11,7 +11,9 @@
             "product": ".NET",
             "release-type" : "sts",
             "support-phase": "preview",
-            "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json"
+            "releases.json": "https://dotnetcli.blob.core.windows.net/dotnet/release-metadata/9.0/releases.json",
```

Comment:
So if I understand you correctly, we should use GH links here?

Reply:
I meant that we should offer both. We should continue using the Azure links to avoid a compat break, but I think the GitHub links would be a good idea. It's very odd that the files are hosted on GH, but it is impossible to get to those files (with the links provided) if you start the `release-index.json`.